src/js/video.js

```diff
@@ -749,5 +749,9 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+Object.defineProperty(videojs, 'TERMINATOR', {
```

Comment:
should this be directly on `videojs`? Maybe namespaced `videojs.middleware.TERMINATOR`?

Reply:
I like the idea of it being namespaced to `videojs.middleware.TERMINATOR`